packages/common/src/lib/common/logger.ts

```diff
@@ -1,28 +1,34 @@
 import winston from 'winston';
 import DailyRotateFile from 'winston-daily-rotate-file';
-import fs from 'fs';
-import path from 'path';
-import os from 'os';
-import * as tools from './tools';
+import fs from 'node:fs';
+import path from 'node:path';
+import os from 'node:os';
+import * as tools from '@/lib/common/tools.js';
 import Transport from 'winston-transport';
 import { LEVEL } from 'triple-beam';
 import deepClone from 'deep-clone';
 import type { Syslog } from 'winston-syslog';
 import type { SeqTransport } from '@datalust/winston-seq';
+import * as url from 'node:url';
+import { createRequire } from 'node:module';
+
+// eslint-disable-next-line unicorn/prefer-module
+const thisDir = url.fileURLToPath(new URL('.', import.meta.url || 'file://' + __dirname));
+
+// eslint-disable-next-line unicorn/prefer-module
+const require = createRequire(import.meta.url || 'file://' + __dirname);
 
 const hostname = tools.getHostName();
 
 let SysLog: typeof Syslog | undefined;
 try {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires
     SysLog = require('winston-syslog').Syslog;
```

Comment:
We cannot use top-level await if we transpile to cjs with https://github.com/AlCalzone/esm2cjs, hence I kept it require calls